jacodb-ets/build.gradle.kts

```diff
@@ -21,32 +21,46 @@ dependencies {
 
 // Example usage:
 // ```
-// ARKANALYZER_DIR=~/dev/arkanalyzer ./gradlew generateTestResources
+// export ARKANALYZER_DIR=~/dev/arkanalyzer
+// ./gradlew generateTestResources
 // ```
 tasks.register("generateTestResources") {
     group = "build"
     description = "Generates test resources from TypeScript files using ArkAnalyzer."
     doLast {
+        println("Generating test resources using ArkAnalyzer...")
+        val startTime = System.currentTimeMillis()
+
         val envVarName = "ARKANALYZER_DIR"
-        val defaultArkAnalyzerDir = "../arkanalyzer"
+        val defaultArkAnalyzerDir = "arkanalyzer"
 
         val arkAnalyzerDir = rootDir.resolve(System.getenv(envVarName) ?: run {
```

Comment:
Should we use `rootDir.resolveSibling` if the system environment variable is not set? Either way, it won't affect the path resolving if the environment variable is set.